client/src/components/RelatedItems/RelatedItems.jsx

```diff
@@ -28,7 +29,9 @@ const RelatedItems = ({productId}) => {
 
   return (
     <div>
-      Related Items
+      {relatedItems.map((item) => {
+        return <RelatedItemsCard item={item}/>
+      })}
```

Comment:
you might want to consider adding a key as a prop so that react doesn't complain about you not having a key (line 33)